pkg/codegen/schema.go

```diff
@@ -66,15 +66,6 @@ func (s *Schema) AddProperty(p Property) error {
 	return nil
 }
 
-func (s Schema) GetAdditionalTypeDefs() []TypeDefinition {
```

Comment:
this will need re-adding, as it's part of the API contract for the package, so this is a breaking change

Reply:
Thanks @jamietanna - I've readded it, but it now has a much simpler implementation because the additional types are tracked when the schema is built up.